app/services/ncr/approval_manager.rb

```diff
@@ -62,7 +62,11 @@ def notify_removed_approvers(original_approvers)
     def ba_6x_approvers
       results = []
 
-      unless work_order.for_whsc_organization?
+      if work_order.for_whsc_organization?
+        # no tier 1
+      elsif work_order.ba_6x_tier1_team?
+        results << Ncr::Mailboxes.ba61_tier1_budgetteam
+      else
```

Comment:
I wanted to make it clear there are 3 distinct cases:

1. the WO has a whsc_organization (so no tier1 approver at all)
1. the WO has a ba_6x special organization (special tier1 approver)
1. neither of the first 2 are true (default tier1 approver)

I can rewrite it as an if/elsif but I was worried that it the business logic would be less clear later.